sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -44,6 +44,7 @@ const int UR_EXT_DEVICE_INFO_FREE_MEMORY = UR_EXT_DEVICE_INFO_END - 13;
 // const int ZER_EXT_DEVICE_INFO_IMAGE_MAX_ARRAY_SIZE =
 //     UR_DEVICE_INFO_IMAGE_MAX_ARRAY_SIZE;
 const int UR_EXT_DEVICE_INFO_MEM_CHANNEL_SUPPORT = UR_EXT_DEVICE_INFO_END - 15;
+const int UR_EXT_DEVICE_INFO_IMAGE_SRGB = UR_EXT_DEVICE_INFO_END - 16;

```

Comment:
@kbenzie: please take it into UR

Reply:
We alreayd have `UR_DEVICE_INFO_IMAGE_SRGB` in [ur_device_info_t](https://oneapi-src.github.io/unified-runtime/core/api.html#ur-device-info-t). I think that should be sufficient.

Reply:
Having checked out this branch I can see `UR_DEVICE_INFO_IMAGE_SRGB` is defined in the `ur_api.h` header and should be used instead of this new symbol.